src/OSPSuite.Core/Converter/v5_5/Converter541To551.cs

```diff
@@ -28,26 +28,26 @@ public Converter541To551(IDimensionFactory dimensionFactory)
          _amountDimension = dimensionFactory.Dimension(Constants.Dimension.AMOUNT);
       }
 
-      public bool IsSatisfiedBy(int version)
-      {
-         return PKMLVersion.V5_4_1.Equals(version);
-      }
+      public bool IsSatisfiedBy(int version) => version == PKMLVersion.V5_4_1;
 
-      public int Convert(object objectToUpdate)
+      public (int convertedToVersion, bool conversionHappened) Convert(object objectToUpdate)
       {
          this.Visit(objectToUpdate);
-         return PKMLVersion.V5_5_1;
+
+         //too many conversions going on in this converted. Assume conversion
+         return (PKMLVersion.V5_5_1, true);
       }
 
-      public int ConvertXml(XElement element)
+      public (int convertedToVersion, bool conversionHappened) ConvertXml(XElement element)
       {
          element.DescendantsAndSelfNamed("ParameterStartValues", "ParameterStartValuesBuildingBlock").Each(ConvertParameterStartValuesElement);
          element.DescendantsAndSelfNamed("MoleculeStartValues", "MoleculeStartValuesBuildingBlock").Each(ConvertMoleculeStartValueElement);
 
          //"ModelCoreSimulation" required for older project
          element.DescendantsAndSelfNamed("Simulation", "ModelCoreSimulation").Each(ConvertSimuationElement);
 
-         return PKMLVersion.V5_5_1;
+         //too many conversions going on in this converted. Assume conversion
+         return (PKMLVersion.V5_5_1, true);
```

Comment:
This is an old one and almost everything is changing. Return true. If we load such an old project, conversion happens the message will be shown even if no conversion happens. No big deal (and again, everything is converted :)